### PR TITLE
use properties/{keyName}/lookups endpoint

### PR DESCRIPTION
**Pull request by andreatamezm**

### CODE REVIEW

**Review by rchristy0 — CHANGES_REQUESTED** [review]
Can you add a test for the null keyName case?

File: src/test/java/com/genability/client/api/service/PropertyServiceTests.java [hunk 1]
```diff
@@ -115,6 +115,30 @@ public void testGetPropertyLookups() {
 		assertEquals("Got results for the wrong propertyKey",
 				targetPropertyKey, restResponse.getResults().get(0).getPropertyKey());
 	}
+
+	@Test
+	public void testGetPropertyLookups_null_keyName() {
+		GetPropertyLookupsRequest request = new GetPropertyLookupsRequest();
+		try {
+			propertyService.getPropertyLookups(request);
```
Comment (rchristy0) [inline src/test/java/com/genability/client/api/service/PropertyServiceTests.java:123]: you can add a fail() here to verify that the try doesn't succeed.

File: src/test/java/com/genability/client/api/service/PropertyServiceTests.java [hunk 2]
```diff
@@ -115,6 +115,30 @@ public void testGetPropertyLookups() {
...
+
+	@Test
+	public void testGetPropertyLookups_null_keyName_nonNull_additionalParams() {
+		GetPropertyLookupsRequest request = new GetPropertyLookupsRequest();
+		request.setSubKeyName("51291");
+		request.setFromDateTime(new DateTime("2014-01-01"));
+		request.setToDateTime(new DateTime("2014-01-02"));
+
+		try {
+			propertyService.getPropertyLookups(request);
```
Comment (rchristy0) [inline src/test/java/com/genability/client/api/service/PropertyServiceTests.java:137]: same here

**Review by rchristy0 — CHANGES_REQUESTED** [review]

**Review by rchristy0 — APPROVED** [review]
